packages/manager/src/features/GlobalNotifications/DatabaseMigrationInfoBanner.tsx

```diff
@@ -0,0 +1,18 @@
+import { Notice, Typography } from '@linode/ui';
+import React from 'react';
+
+export const DatabaseMigrationInfoBanner = () => {
+  return (
+    <Notice important top={10} variant="warning">
+      <Typography fontFamily={(theme) => theme.font.bold} lineHeight="20px">
+        Legacy clusters decommission
+      </Typography>
+      <Typography lineHeight="20px">
+        The legacy database clusters will be available only till the end of{' '}
+        2025. By that time, we’ll migrate your clusters to the new solution. In
+        case of questions regarding the new database clusters or the migration,
+        contact the Support team.
```

Comment:
Agreed that this copy feels a little off from our standard banner messaging. Did tech writing or UX (e.g. @kcosby) have a look at this?

Here's my suggestion for copy revisions:
```suggestion
        Legacy database clusters will only be available until the end of{' '}
        2025. At that time, we’ll migrate your clusters to the new solution. For questions regarding the new database clusters or the migration,
        contact Support.
```